src/ServiceDescription/Shopify-v1.php

```diff
@@ -1899,6 +1899,226 @@
+                    'location'    => 'json',
+                    'type'        => 'string',
+                    'required'    => false,
+                ],
+                'client_details' => [
+                    'description' => 'An object containing information about the client',
+                    'location'    => 'json',
+                    'type'        => 'object',
+                    'required'    => false,
+                ],
+                'closed_at' => [
+                    'description' => 'The date and time when the order was closed',
+                    'location'    => 'json',
+                    'type'        => 'string',
+                    'required'    => false,
+                ],
+                'currency' => [
+                    'description' => 'The three letter code (ISO 4217) for the currency used for the payment',
+                    'location'    => 'json',
+                    'type'        => 'string',
+                    'required'    => false,
+                ],
+                'customer' => [
+                    'description' => 'An object containing information about the customer',
+                    'location'    => 'json',
+                    'type'        => 'object',
+                    'required'    => false,
+                ],
+                'discount_codes' => [
+                    'description' => 'Applicable discount codes that can be applied to the order',
+                    'location'    => 'json',
+                    'type'        => 'array',
+                    'required'    => false,
+                ],
+                'email' => [
+                    'description' => 'The customer\'s email address',
+                    'location'    => 'json',
+                    'type'        => 'string',
+                    'required'    => false,
+                ],
+                'financial_status' => [
+                    'description' => 'The order financial status',
+                    'location'    => 'json',
+                    'type'        => 'object',
+                    'required'    => false,
+                ],
+                'fulfillments' => [
+                    'description' => 'The order fulfillments',
+                    'location'    => 'json',
+                    'type'        => 'array',
+                    'required'    => false,
+                ],
+                'fulfillment_status' => [
+                    'description' => 'The order fulfillment status',
+                    'location'    => 'json',
+                    'type'        => 'string',
+                    'required'    => false,
+                ],
+                'tags' => [
+                    'description' => 'Tags are additional short descriptors',
+                    'location'    => 'json',
+                    'type'        => 'string',
+                    'required'    => false,
+                ],
+                'landing_site' => [
```

Comment:
Are you sure this one is writable? I think Shopify uses the referrer ID to populate it